cluster/gce/gci/configure-helper.sh

```diff
@@ -1703,6 +1703,11 @@ function start-kube-addons {
   if [[ "${ENABLE_NODE_LOGGING:-}" == "true" ]] && \
      [[ "${LOGGING_DESTINATION:-}" == "gcp" ]]; then
     setup-addon-manifests "addons" "fluentd-gcp"
+    local -r fluentd_gcp_yaml="${dst_dir}/fluentd-gcp/fluentd-gcp-ds.yaml"
+    kubectl set resources --dry-run --local -f ${fluentd_gcp_yaml} \
+      --limits=memory=${FLUENTD_GCP_MEMORY_LIMIT} \
+      --requests=cpu=${FLUENTD_GCP_CPU_REQUEST},memory=${FLUENTD_GCP_MEMORY_REQUEST} \
+      -o yaml > ${fluentd_gcp_yaml}
```

Comment:
It worked for me, but I agree this may not be always true. Introduced temporary file.